### PR TITLE
Update Authorization Steps for CSM Operator

### PR DESCRIPTION
**Pull request by shaynafinocchiaro**
# Description
PR to update steps for installing authorization proxy server via csm operator.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/633|

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csmoperator/modules/authorization.md [hunk 1]
```diff
@@ -154,6 +151,15 @@ To deploy the Operator, follow the instructions available [here](../../#installa
    | **cert-manager** | This section configures the enablement of cert-manager. | - | - |
    | enabled | Enable/Disable deployment of cert-manager. Set to false if you already have cert-manager installed. | No | true |
 
+**Optional:**
+To enable reporting of trace data with [Zipkin](https://zipkin.io/), this can be achieved through the `csm-config-params` configMap in the sample CR or dynamically by editing the configMap.
```
Comment (rsedlock1958) [inline content/docs/deployment/csmoperator/modules/authorization.md:155]: Replace "this can  be achieved through the" with "use"

**Review by rsedlock1958 — COMMENTED** [review]
I have reviewed this. One grammatical edit is required.

File: content/docs/deployment/csmoperator/modules/authorization.md (hunk 1 above)
Comment (shaynafinocchiaro) [reply content/docs/deployment/csmoperator/modules/authorization.md]: fixed.

**Review by atye — APPROVED** [review]

**Review by alikdell — APPROVED** [review]

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No grammatical edits are required.

**Pull request merged**